backend/src/test/java/gov/cdc/ReportVision/models/TemplateTest.java

```diff
@@ -0,0 +1,31 @@
+package gov.cdc.ReportVision.models;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.Set;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class TemplateTest {
+
+    @Test
+    public void testTemplateValidations() {
```

Comment:
At some point we can create separate set up method for initializing the validators but looks good for now!